examples/api-examples/create-workbench/api-example.py

```diff
@@ -0,0 +1,421 @@
+                                "secretRef": {
+                                    "name": common_name,
+                                },
+                            },
+                        ],
+                        "image": "image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/s2i-minimal-notebook:2024.2",
+                        "imagePullPolicy": "Always",
+                        "livenessProbe": {
+                            "failureThreshold": 3,
+                            "httpGet": {
+                                "path": f"/notebook/{common_namespace}/{common_name}/api",
+                                "port": "notebook-port",
+                                "scheme": "HTTP",
+                            },
+                            "initialDelaySeconds": 10,
+                            "periodSeconds": 5,
+                            "successThreshold": 1,
+                            "timeoutSeconds": 1,
+                        },
+                        "name": common_name,
+                        "ports": [
+                            {
+                                "containerPort": 8888,
+                                "name": "notebook-port",
+                                "protocol": "TCP",
+                            },
+                        ],
+                        "readinessProbe": {
+                            "failureThreshold": 3,
+                            "httpGet": {
+                                "path": f"/notebook/{common_namespace}/{common_name}/api",
+                                "port": "notebook-port",
+                                "scheme": "HTTP",
+                            },
+                            "initialDelaySeconds": 10,
+                            "periodSeconds": 5,
+                            "successThreshold": 1,
+                            "timeoutSeconds": 1,
+                        },
+                        "resources": { # These resources align with the default Small workbench size
+                            "limits": {
+                                "cpu": "2",
+                                "memory": "8Gi",
+                            },
+                            "requests": {
+                                "cpu": "1",
+                                "memory": "8Gi",
+                            },
+                        },
+                        "volumeMounts": [
+                            {
+                                "mountPath": "/opt/app-root/src",
+                                "name": common_name,
+                            },
+                            {
+                                "mountPath": "/dev/shm",
+                                "name": "shm",
+                            },
+                            { # This is that RHOAI-injected CA bundle - note that it's not specified in the Volumes, but it will be mutated to include this
```

Comment:
Thought the Notebook Controller would also add the volumemount, not just module.